deployment/DB_VERSION

```diff
@@ -1 +1 @@
-79
+80
```

Comment:
Master seems [already at 80](https://github.com/aeternity/epoch/blob/452fcbd65e9db42f81bf222325023e88b2eaad03/deployment/DB_VERSION#L1) - blocking so we remember.

BTW there will be conflicts e.g. https://github.com/aeternity/epoch/pull/1470